core/icons/icon_types.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** The type for a mutator icon. Used for registration and access. */
+export const MUTATOR_TYPE = 'mutator';
```

Comment:
Why do we need these defined here and in the respective file?

The only thing I can think of is that people would be allowed to register their own CommentIcon class for Blockly to use, and maybe it makes sense to provide a string they can use as the type for registration here so they don't have to use the one in `CommentIcon`. But a) if they can provide their own `CommentIcon` it should probably be a subclass of ours, so they could just rely on inheritance b) if developers are supposed to use this string, I think you need to export it from Blockly and I don't see where that's happening.